hoppfish/hwangsae-recorder-user-command.py

```diff
@@ -63,7 +63,79 @@ def post(self, method, edge_id):
             http_ret = 404
         return response, http_ret
 
+class vod(Resource):
+    def get(self, method):
+        http_ret = 201
+        try:
+            obj = dbus.SessionBus().get_object('org.hwangsaeul.Hwangsae1.RecorderAgent',
+                    '/org/hwangsaeul/Hwangsae1/RecorderInterface')
+            dbus_interface = dbus.Interface(obj,
+                    dbus_interface='org.hwangsaeul.Hwangsae1.RecorderInterface')
+
+            ret = ''
+            if method == 'lookup-by-record':
+                record_id = request.args.get('recordId')
+                try:
+                    range_from = int(request.args.get('from'))
+                except:
+                    range_from = 0
+                try:
+                    range_to = int(request.args.get('to'))
+                except:
+                    range_to = 0
+                ret = dbus_interface.LookupByRecord(record_id, range_from, range_to)
+                edge_id = ret[0]
+                file_list = ret[1]
+                ret = {}
+                ret['edgeId'] = edge_id
+                file_list_ret = []
+                for f in file_list:
+                    file_data = {}
+                    file_data['fileId'] = f[0]
+                    file_data['start'] = f[1]
+                    file_data['end'] = f[2]
+                    file_data['size'] = f[3]
+                    file_list_ret.append(file_data)
+                ret['fileList'] = file_list_ret
+                ret = json.dumps(ret)
+
+            elif method == 'lookup-by-edge':
+                edge_id = request.args.get('edgeId')
+                try:
+                    range_from = int(request.args.get('from'))
+                except:
+                    range_from = 0
+                try:
+                    range_to = int(request.args.get('to'))
+                except:
+                    range_to = 0
+                ret = dbus_interface.LookupByEdge(edge_id, range_from, range_to)
+                file_list = ret
+                ret = {}
+                file_list_ret = []
+                for f in file_list:
+                    file_data = {}
+                    file_data['recordId'] = f[0]
+                    file_data['fileId'] = f[1]
+                    file_data['start'] = f[2]
+                    file_data['end'] = f[3]
+                    file_data['size'] = f[4]
+                    file_list_ret.append(file_data)
+                ret['fileList'] = file_list_ret
+                ret = json.dumps(ret)
+
+            response = ret
+
+        except Exception as e:
+            print ('Exception:', e)
+            response = "internal error"
+            http_ret = 404
+        return response, http_ret
+
+        return response, http_ret
+
 api.add_resource(srt, '/api/v1.0/record/<method>/<edge_id>')
+api.add_resource(vod, '/api/v1.0/record/<method>')
```

Comment:
Api is different from document. Api for lookup and file managing service is "/api/v1.0/vod/<method>"

Reply:
Thanks for spotting the error.